### PR TITLE
Document the SQLite utilities usage

### LINKED ISSUE #961: Document migration MySQL <-> SQLite <-> PostgreSQL
Is there a way to migrate from sqlite to mysql? I have started testing with sqlite setting. But now i would like to switch to mysql. How can i do this? 

Is there a possibility / setting / procedure to do this? If not, can you explain me the manual way of migration from sqlite to mysql? 

Thanks

### PR DESCRIPTION
**Pull request by aledeg**
Closes #961

Changes proposed in this pull request:

- Update SQLite utilities documentation

Pull request checklist:

- [ ] clear commit messages
- [ ] code manually tested
- [ ] unit tests written (optional if too hard)
- [x] documentation updated

[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/master/docs/en/developers/04_Pull_requests.md).

There is a CLI command to export from any database to a SQLite database
and also a CLI command to import a SQLite database to any database than
need to be documented.

### CODE REVIEW

File: docs/en/admins/05_Backup.md [hunk 1]
```diff
@@ -49,7 +49,21 @@ You can export your feed list in OPML format either from the web interface, or f
...
 ```
+
+**From any database**  
+You can use the [Command-Line Interface](https://github.com/FreshRSS/FreshRSS/blob/master/cli/README.md) to export your database to a SQLite database file:
+```
+./cli/export-sqlite-for-user.php --user <username> --filename </path/to/db.sqlite>
+```
+> Note that the database filename needs the `sqlite` extension in order to work properly.
+
+You can use again the [Command-Line Interface](https://github.com/FreshRSS/FreshRSS/blob/master/cli/README.md) to import the SQLite database file into your database:
```
Comment (Frenzie) [inline docs/en/admins/05_Backup.md]:
```suggestion
You can use the [Command-Line Interface](https://github.com/FreshRSS/FreshRSS/blob/master/cli/README.md) again to import the SQLite database file into your database:
```

Comment (aledeg) [reply docs/en/admins/05_Backup.md]: fixed

File: docs/en/admins/05_Backup.md [hunk 2]
```diff
@@ -49,7 +49,21 @@ You can export your feed list in OPML format either from the web interface, or f
...
+```
+./cli/export-sqlite-for-user.php --user <username> --filename </path/to/db.sqlite>
+```
+> Note that the database filename needs the `sqlite` extension in order to work properly.
+
+You can use the [Command-Line Interface](https://github.com/FreshRSS/FreshRSS/blob/master/cli/README.md) again to import the SQLite database file into your database:
+```
+./cli/import-sqlite-for-user.php --user <username> --filename </path/to/db.sqlite>
+```
+> Again, note that the database filename needs the `sqlite` extension in order to work properly.
```
Comment (Alkarex) [inline docs/en/admins/05_Backup.md:69]:
```suggestion
> Again, note that the database filename needs the `sqlite` extension in order to work properly.

This process via SQLite is useful for exporting a user fully, for backups, for migrating to another server, for changing database type, or even for attempting to fix database corruptions.
```

**Review by Alkarex — APPROVED** [review]

**Comment by Alkarex** [comment]
It would be nice to update that section as well:
https://github.com/FreshRSS/FreshRSS/blob/master/README.fr.md#sauvegarde

**Comment by aledeg** [comment]
Fixed!

**Review by Alkarex — APPROVED** [review]

**Pull request merged**